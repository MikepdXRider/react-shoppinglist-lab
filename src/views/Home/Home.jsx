import React from 'react';
import ItemForm from '../../components/Forms/ItemForm.jsx';
import ShoppingList from '../../components/ShoppingList/ShoppingList.jsx';
export default function Home() {
    return (
        <main>
            <ShoppingList />            
        </main>
    )
}
