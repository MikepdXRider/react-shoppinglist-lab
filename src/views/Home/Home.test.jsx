import { render } from '@testing-library/react';
import Home from "./Home.jsx";


it('renders home view component', () => {
    const {container} = render(
        <Home />
    )

    expect(container).toMatchSnapshot();
})