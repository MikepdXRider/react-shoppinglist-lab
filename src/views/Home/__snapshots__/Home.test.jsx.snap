// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home view component 1`] = `
<div>
  <main>
    <section>
      <div>
        <form>
          <fieldset>
            <label
              for="new-item"
            >
              New Item: 
            </label>
            <input
              id="new-item"
              required=""
              type="text"
              value=""
            />
            <button
              aria-label="add-new-item"
            >
              Add
            </button>
          </fieldset>
        </form>
      </div>
      <ul>
        <li>
          <article>
            <h1>
              Milk
            </h1>
            <button>
              Delete
            </button>
            <button>
              Edit
            </button>
          </article>
        </li>
        <li>
          <article>
            <h1>
              Burrito
            </h1>
            <button>
              Delete
            </button>
            <button>
              Edit
            </button>
          </article>
        </li>
        <li>
          <article>
            <h1>
              Pesto Sauce
            </h1>
            <button>
              Delete
            </button>
            <button>
              Edit
            </button>
          </article>
        </li>
      </ul>
    </section>
  </main>
</div>
`;
